Personal Access Tokens (#501)
livechat/livechat-public-docs: content/getting-started/authorization/index.mdx

This authorization flow is not yet available.

## Personal Access Tokens

Unlike other authorization flows that let you acquire a token for your app, **Personal Access Tokens (PATs)** are generated per Agent.

> With the use of PATs, you can make requests to the Agent Chat and Configuration API but not to the Customer Chat API.

Reviewer: Pozwala też robić requesty do Reports API, i pewnie gdzieś jeszcze. Generalnie do większości API, nie pisałbym że Customer Chat API jest wykluczone.

Author: Ok. The argument that someone may CTRL + F and find Customer API convinced me. 
Added Reports API.

 

### Step 1: Generate a PAT

> Start in [Developer Console](https://developers.livechatinc.com/console/). Then, go to **Tools -> Personal Access Tokens** and create a new PAT. Remember to add any scopes you might need later.

Reviewer: ```suggestion
Start in [Developer Console](https://developers.livechatinc.com/console/). Then, go to **Tools -> Personal Access Tokens** and create a new token and choose scopes for it.
```
